datahub-classify/src/datahub_classify/infotype_predictor.py

```diff
@@ -24,7 +24,11 @@ def get_infotype_function_mapping(
             logger.warning(f"Configuration is not available for infotype - {infotype}")
         else:
             fn_name = f"inspect_for_{infotype.lower()}"
-            infotype_function_map[infotype] = module_fn_dict[fn_name]
+            if fn_name in module_fn_dict:
+                infotype_function_map[infotype] = module_fn_dict[fn_name]
```

Comment:
`module_fn_dict` - let's make this a more explicit dict in the future. not a huge fan of doing it implicitly like this based on the module/function names